CHANGELOG.md

```diff
@@ -0,0 +1,265 @@
+# Changelog
+
+## [Unreleased]
+
+### Changed
+* Upgrade `level-js` from `~2.2.4` to `~3.0.0` (@vweevers, @greenkeeper)
+* Upgrade `level-packager` from `~2.1.1` to `~4.0.0` (@vweevers, @greenkeeper)
+* Upgrade `memdown` from `~2.0.0` to `~3.0.0` (@vweevers, @greenkeeper)
+* Upgrade `leveldown` optionalDependency from `~3.0.1` to `~4.0.0` (@vweevers, @greenkeeper)
+* Upgrade `airtap` devDependency from `0.0.4` to `0.0.8` (@vweevers, @greenkeeper)
+* Tweak copyright years for less maintenance (@ralphtheninja)
+* Update repository url (@vweevers)
+
+### Added
+* Add `CHANGELOG.md` (@ralphtheninja)
+
+## [3.0.0] - 2018-05-10
+
+### Changed
+* Update README style (@ralphtheninja, @vweevers)
+* Upgrade `level-js` from `~2.1.6` to `~2.2.4` (@ralphtheninja, @greenkeeper)
+* Upgrade `memdown` from `~1.0.0` to `~1.2.4` (@ralphtheninja, @greenkeeper)
+* Upgrade `memdown` from `~1.2.4` to `~2.0.0` (@vweevers)
+* Upgrade `rimraf` from `~2.3.4` to `~2.6.1` (@ralphtheninja, @greenkeeper)
+* Upgrade `tape` devDependency from `~3.0.0` to `~4.8.0` (@ralphtheninja, @greenkeeper)
+* Simplify `npm test` (@vweevers)
+* Use unique name for each test (@vweevers)
+* Use caret for devDependencies (@vweevers)
+* Switch to use `level-packager` (@vweevers)
+* Merge identical disk and memory tests (@vweevers)
+* Refactor browser code and test with `airtap` (@vweevers)
+* Change LICENSE to LICENSE.md (@vweevers)
+* Make arguments optional (@vweevers)
+* Dedup code between node and the browser (@vweevers)
+
+### Added
+* Add node 6 and 8 to Travis (@ralphtheninja)
+* Add node 9 and 10 to Travis (@vweevers)
+* Add `standard` (@vweevers)
+* Test that db is backed by `memdown` (@vweevers)
+* Test custom store using `jsondown` (@vweevers)
+
+### Removed
+* Remove node 0.10 from Travis (@ralphtheninja)
+* Remove `level` dependency (@vweevers)
+* Remove `levelup` dependency (@vweevers)
+* Remove `leveldown` dependency (@vweevers)
+* Remove irrelevant encoding option from test (@vweevers)
+* Remove `testling` from `package.json` (@vweevers)
+
+### Fixed
+* Names in clean tests should be the same (@vweevers)
```

Comment:
Can leave this out, the names were initially the same (before 3.0.0) but when I refactored the test I didn't realize they had to be. There's no change between 2.x and 3.x.